src/lib/config.ts

```diff
@@ -770,6 +770,16 @@ class default_config {
      */
     updatenaglastversion = "1.14.0"
 
+    /**
+     * Time we last checked for an update, milliseconds since unix epoch.
+     */
+    updatelastcheck = 0
```

Comment:
I don't think so.

The number of update-related configuration options has grown to the point where it might be worth using an object to keep it tidier?

```
updates.timeoflastcheck
updates.lastversionnagged
```

etc?

I'm not sure if we're breaking convention by doing that though. I think we've only ever used objects when we wanted people to be able to add extra elements as a kind of pseudo array that we iterate over.

Reply:
`followpagepatterns.{next, prev}` exists. These should be... pretty much entirely internal-only, and we wouldn't be breaking anyone's configs by migrating them?

_goes off to figure out if we have a story for migrating people's config settings..._

Reply:
I don't think anyone will care. Up to you.

Reply:
Decided to namespace them, since I agree that it's a touch dirty, and to not migrate.